frontend/src/metabase-lib/viz/display.ts

```diff
@@ -74,7 +74,7 @@ export const defaultDisplay = (query: Lib.Query): DefaultDisplay => {
       return { display: "bar" };
     }
 
-    if (Lib.isBoolean(column) || Lib.isCategory(column)) {
+    if (Lib.isBoolean(column)) {
```

Comment:
I can believe that there can be a chart with 1 aggregation and 1 boolean breakout.

Reply:
If someone in DashViz has context on that, it could be useful to double check?